packages/component-library/src/Badge/Badge.js

```diff
@@ -0,0 +1,32 @@
+/** @jsx jsx */
+import { jsx, css } from "@emotion/core";
+import PropTypes from "prop-types";
+import { Badge as MaterialBadge } from "@material-ui/core";
+
+const badgeClass = color => css`
+  background-color: #ffffff;
+  span {
+    background-color: ${color};
+  }
+`;
+
+function Badge({ children, value, color }) {
+  return (
+    <MaterialBadge css={badgeClass(color)} badgeContent={value} color="primary">
+      {children}
+    </MaterialBadge>
+  );
+}
+
+Badge.defaultProps = {
+  value: 42,
+  color: "red"
```

Comment:
Let's set this to Civic Medium [colors](https://hackoregon.github.io/civic/?path=/story/design-brand--color-theme)